From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Tue, 3 Nov 2020 22:12:04 +0000
Subject: doxygen warning defang

Do not allow doxygen warnings to derail the build.
This was suggested by Ryan Curtis as a fix for FTBFS on some architectures.
But be that as it may, we really shouldn't allow documentation build
warnings to break the build.
---
 Doxyfile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Doxyfile b/Doxyfile
index 20aaf1e..30dfcf1 100644
--- a/Doxyfile
+++ b/Doxyfile
@@ -80,7 +80,7 @@ HTML_TIMESTAMP         = NO
 #---------------------------------------------------------------------------
 QUIET                  = NO
 WARNINGS               = YES
-WARN_AS_ERROR          = YES
+WARN_AS_ERROR          = NO
 WARN_IF_UNDOCUMENTED   = YES
 WARN_IF_DOC_ERROR      = YES
 WARN_NO_PARAMDOC       = YES
